forge-core/src/main/java/forge/deck/DeckFormat.java

```diff
@@ -105,6 +105,49 @@ public void adjustCMCLevels(List<ImmutablePair<FilterCMC, Integer>> cmcLevels) {
             cmcLevels.add(ImmutablePair.of(new FilterCMC(3, 3), 3));
         }
     },
+    DuelCommander      ( Range.is(99),                         Range.is(0), 1, null,
+            card -> StaticData.instance().getDuelCommanderPredicate().apply(card)
+    ) {
+        private final Set<String> bannedCommanders = ImmutableSet.of(
```

Comment:
Does the banned commander not covered by the formats file? or maybe introduce a text file that reads from the bannedduelcommander.txt so it can be modded by players who wish to use those commanders or can be controlled by the deck conformance toggle if possible.

Reply:
Nothing in the formats file yet for banned as commander, i followed the approach set by tiny leaders above in that file. I agree though it could be cleaned up, may as well add it for both of them